### ISSUE TITLE
Powershell not showing validation string

### ISSUE THREAD

**Issue by Armaldio** [enquirer/enquirer#113]
OS: Windows 10

On cmd, the *validation string* is shown correctly, on PowerShell, it's not shown

![ezgif com-gif-maker](https://user-images.githubusercontent.com/3089715/51652625-52ea8980-1f5e-11e9-9cf9-ba2fff50748d.gif)
![ezgif com-gif-maker 1](https://user-images.githubusercontent.com/3089715/51652627-54b44d00-1f5e-11e9-928d-196f292ad72f.gif)

**Comment by jonschlinkert** [comment]
Thanks for creating the issue! We'll look into it and see what needs to be done to get this fixed. PRs appreciated!

**Comment by Armaldio** [comment]
After some researches, I found that a bug in Powershell display foreground magenta the same as PS blue bg.

https://github.com/nodejs/node/issues/14243
https://github.com/PowerShell/PowerShell/issues/4266

A solution may be to change magenta to another color: https://github.com/enquirer/enquirer/blob/b3d6295e5046976d78d44731a88aebe0fecf2c7d/lib/styles.js#L39
https://github.com/enquirer/enquirer/blob/master/lib/prompt.js#L133

**Comment by jonschlinkert** [comment]
Great to know! Thanks for researching that! We should be able to detect PowerShell automatically to change colors. However, it seems like this would only be “hacking at the leaves” of the problem, since the same problem might occur in other terminals based on the user’s theme. In the spirit of accessibility, perhaps we should also consider allowing the user to change colors using a flag or env variable? Thoughts?

**Comment by Armaldio** [comment]
Yeah, that could be great to have a theme option to pass here: https://github.com/enquirer/enquirer#enquirer-1 for example.
But if you do that, you let the responsibility to the accessibility to the user.

> since the same problem might occur in other terminals based on the user’s theme
I don't think so, here it's clearly a bug in PS. Color should auto adapt based on bg color in most terminals.

My recommendation would be to just change magenta to, say, red. But I get that changing color might impact users already using the library as is.

> We should be able to detect PowerShell automatically to change colors
This might be the most reasonable one.

Your call ;)
